pandas/tests/generic/test_finalize.py

```diff
@@ -38,14 +38,9 @@
     (pd.Series, ([0],), operator.methodcaller("take", [])),
     (pd.Series, ([0],), operator.methodcaller("__getitem__", [True])),
     (pd.Series, ([0],), operator.methodcaller("repeat", 2)),
-    pytest.param(
-        (pd.Series, ([0],), operator.methodcaller("reset_index")),
-        marks=pytest.mark.xfail,
-    ),
+    pytest.param((pd.Series, ([0],), operator.methodcaller("reset_index"))),
```

Comment:
Don't think the `pytest.param` call is needed anymore since it was only necessary to add the `marks` which is now removed

Reply:
(same as below)